resources/application-connector/templates/applications-microfrontend.yaml

```diff
@@ -15,9 +15,13 @@ spec:
       navigationPath: apps
       viewUrl: '/home/settings/apps'
       order: 1
+      settings:
+        readOnly: {{ js .Values.global.applicationManagementDisabled }}
     - label: Application Details
       navigationPath: apps/details
       viewUrl: '/home/settings/apps'
     - label: Application Details
       navigationPath: apps/details/:name
-      viewUrl: '/home/settings/apps/:name'
\ No newline at end of file
+      viewUrl: '/home/settings/apps/:name'
+      settings:
+        readOnly: {{ js .Values.global.applicationManagementDisabled }}
```

Comment:
AFAIK, context is inherited by child nodes, so the setting on list view level should be sufficient